Jenkinsfile

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env groovy
+@Library('apm@current') _
+pipeline {
+  agent { label 'linux && immutable && debian-11' }
+  environment {
+    VAULT_ADDR=credentials('vault-addr')
+    VAULT_ROLE_ID=credentials('apm-vault-role-id')
+    VAULT_SECRET_ID=credentials('apm-vault-secret-id')
+    DOCKER_PASS=credentials('docker-pass-davesys911')
```

Comment:
are these creds temporary? or do we intend to use yours in the long run? can be problematic imo

Reply:
Will be removed in a follow up pr, without these i have no abillity to test e2e locally. Can remove after i see main works successfully on the actual jenkins cluster

Reply:
sounds good to me